operator/cmd/mesh/testdata/manifest-generate/input/ztunnel_tolerations.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: istio-operator
+spec:
+  profile: ambient
+  components:
+    ztunnel:
+      enabled: true
+      k8s:
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
+          limits:
+            cpu: 200m
+            memory: 200Mi
+        tolerations:
+        - key: "example-key"
+          operator: "Exists"
+          effect: "NoSchedule"
```

Comment:
Would this change anything in the output?

Reply:
It should set the tolerations and overwrite the default